lib/command.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Command from '@oclif/command';
+
+export default abstract class extends Command {
+
+	/**
+	 * When set to true, command will be listed in `help`,
+	 * otherwise listed in `help --verbose` with secondary commands.
+	 */
+	public static primary = false;
+}
```

Comment:
@pdcastro  Well this base class will grow significantly, as we will need to add support for things like capitano's `root` and `permission` concepts here. (I realised that `scan` requires the root concept before it can be merged, so was looking at adding that now).

IMO classes should usually be in their own file (unless they are trivial).  And since this is the base class for all of the commands (and so infrastructural) I don't think it should be hidden in a util module's file.